bitnami/tensorflow-resnet/templates/NOTES.txt

```diff
@@ -27,4 +27,10 @@
 
 2. Test the server with a sample image.
 
-  docker run --rm -it bitnami/tensorflow-resnet bash -c "curl -Lo /tmp/cat.jpg https://tensorflow.org/images/blogs/serving/cat.jpg && resnet_client_cc --server_port=$APP_HOST:$APP_PORT --image_file=/tmp/cat.jpg"
\ No newline at end of file
+  docker run --rm -it bitnami/tensorflow-resnet bash -c "curl -Lo /tmp/cat.jpg https://tensorflow.org/images/blogs/serving/cat.jpg && resnet_client_cc --server_port=$APP_HOST:$APP_PORT --image_file=/tmp/cat.jpg"
+{{- if and (contains bitnami/ .Values.image.repository) (not (regexMatch -r\d+$ .Values.image.tag)) }}
```

Comment:
this specific chart does not have `.Values.image.` but `.Values.server.image` and `.Values.client.image.`

Reply:
Done